sdk-lib/src/main/java/cash/z/ecc/android/sdk/SdkSynchronizer.kt

```diff
@@ -64,6 +64,7 @@ import cash.z.ecc.android.sdk.model.TransactionPool
 import cash.z.ecc.android.sdk.model.TransactionRecipient
 import cash.z.ecc.android.sdk.model.TransactionSubmitResult
 import cash.z.ecc.android.sdk.model.UnifiedSpendingKey
+import cash.z.ecc.android.sdk.model.UnifiedAddressRequest
```

Comment:
Sort order is wrong, which will probably cause a lint error:
```suggestion
import cash.z.ecc.android.sdk.model.UnifiedAddressRequest
import cash.z.ecc.android.sdk.model.UnifiedSpendingKey
```